src/wp-admin/includes/image.php

```diff
@@ -742,7 +475,7 @@ function _wp_make_additional_mime_types( $new_mime_types, $file, $image_meta, $a
 }
 
 /**
- * Generates attachment meta data and create image sub-sizes for images.
+ * Generate attachment meta data and create image sub-sizes for images.
```

Comment:
Not a big deal, but the third-person s here is more true to the core documentation standards, so I'd argue no need to revert this, throughout the PR.
```suggestion
 * Generates attachment meta data and create image sub-sizes for images.
```

Reply:
Sure, we can leave the doc block improvements!